### DISCUSSION TITLE
Interface language

### DISCUSSION THREAD

**Discussion by gate546** [Koenkk/zigbee2mqtt#20442] [General]
First of all let me say that I am enthusiastic about this product, I recently migrated from another system that gave me many problems with various disconnections trouble but after I switched to zigbee2mqtt all my problems disappeared.
So I like to congratulate with the developers and all those who collaborate with this excellent initiative.
Now I'll try explain my problem:
My Windows PC language is English, my Homeassistant language is English but, for some unknown strange reasons, the Z2Mqtt frontend language is in Italian.
I suppose it's because I'm in Italy and Z2Mqtt perhaps refers to some Windows system variable, for example the keyboard or localization.
I would like to change the Z2Mqtt language to English, how can I do it?
I looked at all the documentation I found around and also in the various forums but I didn't find a any solution.
Can someone help me?
Thank you 
Dommy

**Comment by Koenkk** [comment]
There should be a language flag next to the "permit join" button in the header.

**Reply by gate546** [reply]
Correct, trouble solved.
My difficulty arose from the fact that nowhere I found documented this setup .
Moreover, it is so evident, right in the foreground, straight in the eye, but I hadn't seen it.
Many years ago Mr. Edgar Allan Poe wrote: "The best place to hide anything is in plain sight"
Thank you very much.
Dommy
